common/src/main/java/org/dash/wallet/common/ui/enter_amount/AmountView.kt

```diff
@@ -43,6 +46,7 @@ class AmountView(context: Context, attrs: AttributeSet) : ConstraintLayout(conte
         .noCode().minDecimals(2).optionalDecimals()
 
     private var onCurrencyToggleClicked: (() -> Unit)? = null
+    private val analytics = FirebaseAnalyticsServiceImpl.getInstance()
```

Comment:
@ClaudeHangui it's better to use Hilt for injecting the analytics service. `getInstance()` method was added when we didn't have DI in the wallet.